client/session.go

```diff
@@ -78,16 +77,22 @@ const (
 type FetchBlocksMetadataEndpointVersion int
 
 const (
+	// FetchBlocksMetadataEndpointDefault represents to use the default fetch blocks metadata endpoint
+	FetchBlocksMetadataEndpointDefault FetchBlocksMetadataEndpointVersion = iota
 	// FetchBlocksMetadataEndpointV1 represents v1 of the fetch blocks metadata endpoint
-	FetchBlocksMetadataEndpointV1 FetchBlocksMetadataEndpointVersion = iota
+	FetchBlocksMetadataEndpointV1
 	// FetchBlocksMetadataEndpointV2 represents v2 of the fetch blocks metadata endpoint
-	FetchBlocksMetadataEndpointV2
+	FetchBlocksMetadataEndpointV2 = FetchBlocksMetadataEndpointDefault
```

Comment:
I can't unfortunately since the default needs to be a zero value (as there's a lot of place where it's not set by default and we want the default to actual be the new endpoint).